spec/system/admin/admin_bans_or_warns_user_spec.rb

```diff
@@ -1,75 +0,0 @@
-require "rails_helper"
```

Comment:
These specs were working because the `hidden` modal div was technically on the page and reachable by the test. They break now because the appropriate form actions are only attached in JS when the modal opens.

The majority of the specs were already being tested in Cypress, so I've just enhanced those Cypress tests to cover the rest of the test cases here, and removed this file.